lumicks/pylake/tests/test_correlated_stack.py

```diff
@@ -105,11 +105,11 @@ def test_correlation(shape):
     with pytest.raises(AssertionError):
         cc["40ns":"70ns"].downsampled_over(stack[0:1].timestamps)
 
-    assert (stack[0].raw.start == 10)
-    assert (stack[1].raw.start == 20)
-    assert (stack[1:3][0].raw.start == 20)
-    assert (stack[1:3].raw[0].start == 20)
-    assert (stack[1:3].raw[1].start == 30)
+    assert (stack[0]._get_frame(0).start == 10)
+    assert (stack[1]._get_frame(0).start == 20)
+    assert (stack[1:3]._get_frame(0).start == 20)
+    assert (stack[1:3]._get_frame(0).start == 20)
+    assert (stack[1:3]._get_frame(1).start == 30)
```

Comment:
The deprecation message talks about using the public API, but `_something` is deemed private API, no? Shouldn't the tests also use the public API?